applications/crossbar/src/modules/cb_conferences.erl

```diff
@@ -222,8 +234,20 @@ create_conference(Context) ->
 %% @end
 %%--------------------------------------------------------------------
 -spec load_conference(ne_binary(), cb_context:context()) -> cb_context:context().
-load_conference(DocId, Context) ->
-    crossbar_doc:load(DocId, Context, ?TYPE_CHECK_OPTION(<<"conference">>)).
+load_conference(ConfId, Context) ->
+    Realm = kz_util:get_account_realm(cb_context:account_id(Context)),
+    Confs = get_conferences(Realm, ConfId),
+    case participants_info(Confs) of
+        {error, Error} -> cb_context:add_system_error(Error, Context);
+        Participants ->
+            ConfCtx = crossbar_doc:load(ConfId, Context, ?TYPE_CHECK_OPTION(<<"conference">>)),
+            ConfInfo = [
```

Comment:
@jamesaimonetti It's supposed to be an object with 3 k/v, and it is (http returns proper object).